src/status_im/contexts/wallet/add_account/create_account/events.cljs

```diff
@@ -71,3 +71,25 @@
                        (cske/transform-keys transforms/->kebab-case-keyword derived-address)))}))
 
 (rf/reg-event-fx :wallet/get-derived-addresses-success get-derived-addresses-success)
+
+;; Import private key
+
+(rf/reg-event-fx
+ :wallet.import-private-key/set-private-key
```

Comment:
I like the qualification approach here but we haven't used this elsewhere yet. No strong opinion but for consistency it would be good if some other devs gave some input on this approach .

Reply:
I really like it too, I'd ask for @ilmotta's or @flexsurfer's opinion

Reply:
@J-Son89, @ibrkhalil, @OmarBasem, @Rende11 We had a discussion long ago when the codebase had tons of inconsistent names in event IDs. I remember we reached a _soft_ agreement about this recommendation:

> Add logical contexts to event keywords, when necessary.
>
> -- https://github.com/status-im/status-mobile/issues/13996

The examples were exactly as the one used in this PR, i.e. add a period to contextualize the qualification/namespace part of the keyword.

I checked now and, unfortunately, this recommendation never made into our guidelines. I'm taking note of this point and will try to update our guidelines soon.

-----

If we want to contextualize some wallet event IDs, we better do that consistently and probably in a separate PR as you guys suggested.